osutil/disks/disks_linux.go

```diff
@@ -47,6 +48,10 @@ var diskFromMountPoint = func(mountpoint string, opts *Options) (Disk, error) {
 	return diskFromMountPointImpl(mountpoint, opts)
 }
 
+var abstractCalculateLastUsableLBA = func(device string) (uint64, error) {
```

Comment:
@valentindavid the mac OS build of the `snap` command no longer builds:
```
Error: osutil/disks/mockdisk.go:412:9: undefined: abstractCalculateLastUsableLBA
```

you may need to put this definition into `disks.go` instead of here in disks_linux.go